systemtests/src/main/java/io/enmasse/systemtest/messaginginfra/resources/MessagingAddressResourceType.java

```diff
@@ -70,4 +72,13 @@ public static MessagingAddressCondition getCondition(List<MessagingAddressCondit
         }
         return null;
     }
+
+    public static int getPort(String protocol, MessagingEndpoint endpoint) {
```

Comment:
MessagingAddressResourceType seems like the wrong place for this, put it in MessagingEndpointResourceType.

Reply:
Wait, we already have this method, but written differently. I'll remove this and change tests to use that one in EndpointResource.